docs/devices/9290012573A.md

```diff
@@ -17,7 +17,7 @@ pageClass: device-page
 |-----|-----|
 | Model | 9290012573A  |
 | Vendor  | Philips  |
-| Description | Hue white and color ambiance E26/E27/E14 |
+| Description | Hue white and color, E27 socket, 9.5 watt power draw, 806 lumen |
```

Comment:
I believe not all models use the E27 socket so I propose to leave as is

Reply:
You may be right, unfortunately this 9290012573A bulb with E27 is the only Hue bulb that I possess.
I primarily thought think this ID is E27-only because 90%+ of all images when searching for it in google image search are of E27 bulbs.

Reply:
What is the model ID of yours? Can be found in the data/database.db

Reply:
I can unfortunately not pair it because zigbee2mqtt crashes every time, I think the culprit here is my Conbee2, but I'm not sure. This MR was just to share my findings that Gen4 Hue bulbs don't have Bluetooth which I still think should be included in the doc